entries/QUnit.only.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="../entries2html.xsl" ?>
+<entry type="method" name="QUnit.only">
+	<title>QUnit.only()</title>
+	<signature>
+		<argument name="name" type="String">
+			<desc>Title of unit being tested</desc>
+		</argument>
+		<argument name="callback">
+			<desc>Function to close over assertions</desc>
+			<type name="Function">
+				<argument name="assert" type="Assert"></argument>
+			</type>
+		</argument>
+	</signature>
+	<desc>
+		Adds a test to be exclusively ran
+	</desc>
+	<longdesc>
+    <p>Use this method to focus your test suite on a specific test.</p>
+    <p>Note, that if more than one <code>QUnit.only</code> is present only the first instance will run.</p>
```

Comment:
missing tabs instead of regular space.

I miss some explanation on the first paragraph saying `QUnit.only` will tell QUnit to ignore the other tests.